prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display:flex;
+      flex-wrap: wrap;
+      padding: 20px;
+    }
+    .card-container {
+      flex-basis:.166;
```

Comment:
While this will work it isn't exact and it is harder for humans to read. It is better to use the `calc` function, for example:

```CSS
.card-container {
  flex-basis: calc(100% / 6);
}
```

In the above code it is clearer that we are making a column that is 1 6th of the full width.